.github/workflows/build_infra_images_cache.yml

```diff
@@ -31,26 +31,21 @@ jobs:
     if: github.repository == 'apache/spark'
     runs-on: ubuntu-latest
     steps:
-      -
-        name: Checkout Spark repository
+      - name: Checkout Spark repository
         uses: actions/checkout@v2
         with:
           fetch-depth: 0
```

Comment:
```suggestion
```
This should be remove, now only checkout the master branch. Will add branches infra image later, I think